servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/ProxyConnectLBHttpConnectionFactory.java

```diff
@@ -107,12 +102,12 @@ private Single<? extends FilterableStreamingHttpConnection> createConnection(
                         .andThen(new HttpClientChannelInitializer(
                                 getByteBufAllocator(executionContext.bufferAllocator()), h1Config, closeHandler)),
                 observer, h1Config.headersFactory(), connectAddress)
-                .flatMap(ch -> handshake(ch, observer))
+                .flatMap(ProxyConnectLBHttpConnectionFactory::handshake)
                 .flatMap(protocol -> finishConnectionInitialization(protocol, channel, closeHandler, observer))
                 .onErrorMap(cause -> handleException(cause, channel));
     }
 
-    private static Single<String> handshake(final Channel channel, final ConnectionObserver connectionObserver) {
```

Comment:
I'm not sure if this is planned for future use but I just removed it for now so we don't forget

Reply:
just a leftover from intermediate changes, thanks